public/src/app.js

```diff
@@ -6,9 +6,11 @@ import { LocalStorage } from "./utils/LocalStorage.js";
 import { userList } from "./components/userList.js";
 import { initUsername } from "./utils/initUsername.js";
 import { chatMessages } from "./components/chatMessages.js";
+import { KeyManager } from "./utils/KeyManager.js";
 
 const app = (s) => {
   const socket = io.connect("http://localhost:3000");
+  const keysPressed = new KeyManager(s);
```

Comment:
the `KeyManager` takes in the instance of the `p5` library, so we can do p5 things within it - in this case the `p5.save()` function for screenshotting.